data/en/cfadmin.json

```diff
@@ -26,8 +26,8 @@
 	],
 	"examples": [
 		{
-			"title":"Below you will find all the actions sorted alphabetically embedded in examples for them:",
-			"description":"These are examples provided by Lucee for the cfadmin actions in alphabetical order",
+			"title":"Create an admin collection",
+			"description":"",
 			"code":"<cfadmin\n
 			    action=\"collection\"\n
```

Comment:
Unfortunately JSON does not support visual multiline strings. Only `\n` work and works properly in single-line string when you output it.

Reply:
yes i know i am working on it